pkg/stanza/operator/input/windows/input.go

```diff
@@ -120,25 +192,33 @@ func (i *Input) readToEnd(ctx context.Context) {
 
 // read will read events from the subscription.
 func (i *Input) read(ctx context.Context) int {
-	events, err := i.subscription.Read(i.maxReads)
-	if err != nil {
-		i.Logger().Error("Failed to read events from subscription", zap.Error(err))
-		return 0
-	}
+	totalEvents := 0
+	for _, subscription := range i.subscriptions {
+		events, err := subscription.Read(i.maxReads)
+		if err != nil {
+			i.Logger().Error("Failed to read events from subscription", zap.String("server", subscription.Server), zap.Error(err))
+			return 0
```

Comment:
to keep the current verbosity all server names, for which it failed, could be collected and a single error message generated

Reply:
Thanks for the feedback. I've updated the code to ensure that if it fails for one of the remote subscriptions, it will log a message for that specific server but keep trying the others. Additionally, I've made sure to collect all failed server names and generate a single error message to maintain the current verbosity. 

I tested this out with a valid and invalid credential, so now the collector will continue to process logs from other sources even if one fails, while still logging errors for each specific failure. 👍